Library/ViewModels/DiscoveryViewModel.swift

```diff
@@ -176,6 +180,16 @@ DiscoveryViewModelOutputs {
     self.viewWillAppearProperty.value = animated
   }
 
+  private static func toString(_ dictionary: [String: String]?) -> String {
```

Comment:
`abExperiments` is optional on `Config` so we don't need to return the default empty string. Also - wouldn't it be better to make `Experiment` conform to `Codable` and then we could just use  `JSONDecoder`'s `KeyedDecodingContainer` to encode/decode this? Maybe I'm missing something.